pkg/ensurance/collector/cadvisor/cadvisor_linux.go

```diff
@@ -45,8 +45,7 @@ type CadvisorCollector struct {
 	latestContainersStates map[string]ContainerState
 }
 
-func NewCadvisor(podLister corelisters.PodLister) *CadvisorCollector {
-
+func NewCadvisorManager() cmanager.Manager {
```

Comment:
"github.com/google/cadvisor/manager" can not build succfullly without linux,
so cmanager.Manager can not be the output parameters of function.
you can refer to。
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cadvisor/cadvisor_unsupported.go